cloudconfig/podcfg/podcfg.go

```diff
@@ -226,13 +227,27 @@ func (cfg *ControllerPodConfig) GetControllerImagePath() string {
 	return GetJujuOCIImagePath(cfg.Controller.Config, cfg.JujuVersion)
 }
 
-// GetJujuOCIImagePath returns jujud oci image path.
+// GetJUjuDbOCIImagePath returns the juju-db oci image path.
+func (cfg *ControllerPodConfig) GetJujuDbOCIImagePath() string {
+	imageRepo := cfg.Controller.Config.CAASImageRepo()
+	if imageRepo == "" {
+		imageRepo = jujudOCINamespace
+	}
+	return fmt.Sprintf("%s/%s", imageRepo, jujudbOCIName)
+}
+
+// GetJujuOCIImagePath returns the jujud oci image path.
 func GetJujuOCIImagePath(controllerCfg controller.Config, ver version.Number) string {
+	// First check the deprecated "caas-operator-image-path" config.
 	imagePath := controllerCfg.CAASOperatorImagePath()
-	if imagePath == "" {
-		imagePath = fmt.Sprintf("%s/%s:%s", jujudOCINamespace, jujudOCIName, ver.String())
+	if imagePath != "" {
+		return imagePath
+	}
+	imageRepo := controllerCfg.CAASImageRepo()
```

Comment:
should `CAASImageRepo` return the whole image path rather than only repo name?
So the controller config will have version specified.